btclightclient/btc_light_client_store.go

```diff
@@ -48,6 +49,11 @@ func NewMemStore() *MemStore {
 	}
 }
 
+func (s *MemStore) IsBlockPresent(h chainhash.Hash) bool {
+	_, ok := s.lightBlockByHashMap[h]
```

Comment:
IsBlockPresent can use `LightBlockByHash` API to check block hash `h` exists in storage. 
```golang
s.LightBlockByHash(h) != nil
```